File: csharp-selenium-nunit-reqnroll/SauceDemo.UiTests/Pages/CheckoutCompletePage.cs
using OpenQA.Selenium;

namespace SauceDemo.UiTests.Pages;

public sealed class CheckoutCompletePage : BasePage
{
    private static readonly By CompleteHeader = By.CssSelector("[data-test='complete-header']");

    public CheckoutCompletePage(IWebDriver driver) : base(driver)
    {
    }

    public string GetConfirmationMessage() => TextOf(CompleteHeader);
}

File: java-selenium-junit5-cucumber/src/test/java/com/saucedemo/uitests/pages/CheckoutCompletePage.java
package com.saucedemo.uitests.pages;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

public final class CheckoutCompletePage extends BasePage {

    private static final By COMPLETE_HEADER = By.cssSelector("[data-test='complete-header']");

    public CheckoutCompletePage(WebDriver driver) {
        super(driver);
    }

    public String confirmationMessage() {
        return textOf(COMPLETE_HEADER);
    }
}
